barretenberg/cpp/src/barretenberg/vm/avm/generated/flavor.hpp

```diff
@@ -121,6 +121,9 @@ class AvmFlavor {
     using VerifierCommitmentKey = AvmFlavorSettings::VerifierCommitmentKey;
     using RelationSeparator = AvmFlavorSettings::RelationSeparator;
 
+    // indicates when evaluating sumcheck, edges must be extended to be MAX_TOTAL_RELATION_LENGTH
+    static constexpr bool USE_SHORT_MONOMIALS = false;
```

Comment:
Please port this change in bb-pilcom/bb-pil-backend/templates/flavor.hpp.hbs
Alternatively, you can ping AVM team and we can do it.

Note that the recursive flavor .hpp counterpart is not generated (therefore no change there)

Reply:
just updated flavor.hpp.hbs